metadata-ingestion/src/datahub/ingestion/source/redshift/query.py

```diff
@@ -443,3 +448,53 @@ def list_copy_commands_sql(
             start_time=start_time.strftime(redshift_datetime_format),
             end_time=end_time.strftime(redshift_datetime_format),
         )
+
+    @staticmethod
+    def get_temp_table_clause(table_name: str) -> List[str]:
+        if table_name.startswith("#"):
+            return [
+                f"{RedshiftQuery.CREATE_TEMPORARY_TABLE_HASH_CLAUSE}{table_name[1:]}"
+            ]
+
+        return [
+            f"{RedshiftQuery.CREATE_TEMP_TABLE_CLAUSE} {table_name}",
+            f"{RedshiftQuery.CREATE_TEMPORARY_TABLE_CLAUSE} {table_name}",
+        ]
+
+    @staticmethod
+    def _like_statement(column_name: str, value: str) -> str:
+        return f"{column_name} ILIKE '{value}%'"
+
+    @staticmethod
+    def temp_table_ddl_query(tables: List[str]) -> str:
```

Comment:
We should already see most scan calls from our queries on stl_scan and stl_insert. Is it possible to avoid this query and instead handle queries related to temp tables as we see them?